src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerbefund/VirologischerBefundCompositionConverter.java

```diff
@@ -0,0 +1,51 @@
+package org.ehrbase.fhirbridge.ehr.converter.specific.virologischerbefund;
+
+import org.ehrbase.fhirbridge.ehr.converter.ConversionException;
+import org.ehrbase.fhirbridge.ehr.converter.generic.ObservationToCompositionConverter;
+import org.ehrbase.fhirbridge.ehr.opt.virologischerbefundcomposition.VirologischerBefundComposition;
+import org.ehrbase.fhirbridge.ehr.opt.virologischerbefundcomposition.definition.FallidentifikationCluster;
+
+import org.hl7.fhir.r4.model.Observation;
+import org.springframework.lang.NonNull;
+import org.hl7.fhir.r4.model.DiagnosticReport;
+
+public class VirologischerBefundCompositionConverter extends ObservationToCompositionConverter<VirologischerBefundComposition> {
+
+        @Override
+        public VirologischerBefundComposition convertInternal(@NonNull Observation resource) {
+
+            VirologischerBefundComposition composition = new VirologischerBefundComposition();
+
+            VirologischerBefundBundle virologischerBefundBundle = new VirologischerBefundBundle(resource);
+            Observation observation = virologischerBefundBundle.getVirologischerBefund();
+
+            DiagnosticReport diagnosticReport = virologischerBefundBundle.getDiagnosticReport();
+
+            observation.setSpecimenTarget(virologischerBefundBundle.getSpecimen());
+
+            mapStatus(composition, diagnosticReport);
+            mapFallIdentifikation(composition, diagnosticReport);
+            composition.setBefund(new VirologischerBefundObservationConverter().convert(observation));
+
+            return composition;
+        }
+
+    private void mapStatus(VirologischerBefundComposition composition, DiagnosticReport diagnosticReport) {
+        String codeString = diagnosticReport.getStatusElement().toString();
+        composition.setStatusValue(codeString);
+    }
+
+    private void mapFallIdentifikation(VirologischerBefundComposition composition, DiagnosticReport diagnosticReport) {
+        FallidentifikationCluster fallidentifikationCluster = new FallidentifikationCluster();
+        if (!diagnosticReport.hasEncounter()){
```

Comment:
As far as i can see in the Profile this is not required, therefore if not existent do not map it. 
Also is Encounter the right field to be mapped for Fallidentifkation, read the descriptions for both. 
Im not sure, your the mapper just double check if its correct.

Reply:
Is now an Optional.
DiagnosticReport.encounter is the correct value to map for Fallidentifikation for this Template after my table. 
https://github.com/ehrbase/fhir-bridge/pull/392/files#diff-7946f6e29b883aeca8c67cd3043f2779692897e990650ff6d5216a3b6b27e0a6